runtime/types.ts

```diff
@@ -20,7 +20,7 @@ export enum MetadataFormulaType {
   SyncGetDisplayUrl = 'SyncGetDisplayUrl',
   SyncGetTableName = 'SyncGetTableName',
   SyncGetSchema = 'SyncGetSchema',
-  PropertyAutocomplete = 'PropertyAutocomplete',
+  PropertyOptions = 'PropertyOptions',
```

Comment:
Will changing the enum value break any existing Packs using this? Fine to break things in an alpha, but would want to give a heads up.

Reply:
Existing `autocomplete` values will stop working for alpha testers, although not specifically from this enum change.

Packs should still run but will start behaving as if no `autocomplete` value was set. Makers can get autocomplete in their packs working again just be renaming to `options` and setting `codaType: ValueHintType.SelectList` on the property.